### ISSUE TITLE
Missing and erroneous documentation for iron-a11y-keys

### ISSUE THREAD

**Issue by barakhub** [PolymerElements/iron-a11y-keys#9]
I think there are at least two issues with the documentation in https://elements.polymer-project.org/elements/iron-a11y-keys. Consider the following example:

``` html
<iron-a11y-keys target="{{}}" keys="up down left right" on-keys-pressed="{{arrowHandler}}"></iron-a11y-keys>
```

First - it seems that the event syntax - `{{arrowHandler}}` is taken from Polymer 0.5. Event handlers are now specified without curly braces.

Second - there is no explanation at all about the `target` attribute. It appears 5 times on the page, all using `{{}}`, which I presume means the entire document (or custom element?). How can one specify anything else?

**Comment by kaycebasques** [comment]
@tjsavage, I can take this.

**Comment by tjsavage** [comment]
Absolutely - apologies for the delay!

**Comment by ebidel** [comment]
Any update on this? I just ran into the outdated docs myself.

**Comment by kaycebasques** [comment]
I've submitted a PR on the docs but am having trouble landing a review. @ebidel help?

**Comment by kaycebasques** [comment]
Fixed in #11.
